app/javascript/components/widgets/widgets/forest-change/tree-loss-plantations/selectors.js

```diff
@@ -17,20 +18,33 @@ const getSentence = state => state.config && state.config.sentence;
 
 // get lists selected
 export const parseData = createSelector(
-  [getLoss, getTotalLoss, getSettings],
-  (loss, totalLoss, settings) => {
-    if (!loss || !totalLoss) return null;
+  [getLossPlantations, getTotalLoss, getSettings],
+  (lossPlantations, totalLoss, settings) => {
+    if (!lossPlantations || !totalLoss) return null;
     const { startYear, endYear } = settings;
     const totalLossByYear = groupBy(totalLoss, 'year');
     return uniqBy(
-      loss.filter(d => d.year >= startYear && d.year <= endYear).map(d => ({
-        ...d,
-        outsideAreaLoss: totalLossByYear[d.year][0].area - d.area,
-        areaLoss: d.area || 0,
-        totalLoss: totalLossByYear[d.year][0].area || 0,
-        outsideCo2Loss: totalLossByYear[d.year][0].emissions - d.emissions,
-        co2Loss: d.emissions || 0
-      })),
+      lossPlantations
+        .filter(d => d.year >= startYear && d.year <= endYear)
+        .map(d => {
+          const groupedPlantations = groupBy(lossPlantations, 'year')[d.year];
+          const summedPlatationsLoss =
+            groupedPlantations && sumBy(groupedPlantations, 'area');
+          const summedPlatationsEmissions =
+            groupedPlantations && sumBy(groupedPlantations, 'emissions');
+
+          const returnData = {
+            ...d,
+            outsideAreaLoss:
+              totalLossByYear[d.year][0].area - summedPlatationsLoss,
```

Comment:
I know this was there before but it concerns me. As it is reused a few times can we declare `totalLossByYear[d.year][0]` as a variable above and then access the keys here. Something like this:

```
const totalLossForYear = (totalLossByYear[d.year] && totalLossByYear[d.year][0]) || {};
```